packages/desktop/components/drawers/network-config/drawers/components/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as NetworkCard } from './NetworkCard.svelte'
+export { default as NetworkStatusPill } from './NetworkStatusPill.svelte'
```

Comment:
I think this should be a shared component, within the `shared/components/atoms/pills` folder.